10_apply_fixation_and_blink_detection_to_gaze.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<p><s>- [...] detect blinks an fixations an how to export data.</s></p><p>+ [...] detect blinks and fixations an<strong>d</strong> how to export data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pupil-labs/pupil-tutorials/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>